src/chrome/content/rules/Reprieve.org.uk.xml

```diff
@@ -6,6 +6,8 @@
 	r: connection refused
 	s: self-signed certificate
 	t: timeout on https
+
+        r: act.reprieve.org.uk (gives 404 error)
```

Comment:
You need to use a custom test url, like `http://act.reprieve.org.uk/page/s/join`